kaltura_admin.py

```diff
@@ -34,74 +38,100 @@ def get_kaltura_session(user_id, partnerID, adminkey):
     return client
 
 
-"""
-Get the configuration file
-"""
-parser = argparse.ArgumentParser(description='Generate Kaltura KMC admin list')
-parser.add_argument('env', type=str, help='env json file with kmc settings')
-args = parser.parse_args()
-if not os.path.isfile(args.env):
-    log.warn("env.json file does not exist. Please provide file path.")
-    sys.exit()
+def get_kmc_admin_list(kaltura_user_id, kmcs):
+    """
+    loop through all Kaltura Management Consoles
+    looking for people with admin roles
+    """
+    column_names = ['kmc_name', 'user_id',
+                    'first_name', 'last_name', 'role_name']
+    user_df = pd.DataFrame(columns=column_names)
+    for kmc_name in kmcs:
+        kmc = kmcs[kmc_name]
+        partnerID = kmc['partnerID']
+        adminkey = kmc['adminkey']
+        client = get_kaltura_session(kaltura_user_id, partnerID, adminkey)
+        filter = KalturaUserFilter()
+        filter.isAdminEqual = KalturaNullableBoolean.TRUE_VALUE
+        filter.loginEnabledEqual = KalturaNullableBoolean.TRUE_VALUE
+        pager = KalturaFilterPager()
+        pager.pageSize = 500
+        result = client.user.list(filter, pager)
+        log.info(
+            f'''Retriving kmc: {kmc_name} Total Records: {result.totalCount}''')
+        for item in result.objects:
+            if item.roleNames == "Publisher Administrator" or item.roleNames == "Content Manager":
+                # remove @xxxxx
+                if '@' in item.id:
```

Comment:
nitpick: This if/else isn't necessary since the code in the if will work fine with the else case.

Reply:
rewrote the if-else statement